File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.java
import java.util.*;
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        int m = nums1.length;
        int n = nums2.length;
        int x = m + n;
        int[] merge = new int[x];
        for (int i = 0; i < m; i++) {
            merge[i] = nums1[i];
        }
        for (int j = 0; j < n; j++) {
            merge[m + j] = nums2[j];
        }
        Arrays.sort(merge);
        if (x % 2 != 0) {
            return merge[x / 2]; 
        } else {
            return (merge[x / 2] + merge[(x / 2) - 1]) / 2.0;
        }
    }
}

File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        int m=nums1.size();
        int n=nums2.size();
        int s=n+m;
        vector<int>nums3(s);
        for(int i=0;i<m;i++)
        {
            nums3[i]=nums1[i];
        }
        for(int j=0;j<n;j++)
        {
            nums3[m+j]=nums2[j];
        }
        sort(nums3.begin(),nums3.end());
        if(s%2==0)
        {
            return (nums3[s/2]+nums3[(s/2)-1])/2.0;
        }
        else
        {
            return nums3[s/2];
        }    
    }
};